apps/payments/next/app/layout.tsx

```diff
@@ -1,9 +1,10 @@
-import ExampleHeader from '../components/ExampleHeader';
+import Image from 'next/image';
+import firefoxLogo from '../images/firefox-logo-combined.svg';
 import './global.css';
 
 export const metadata = {
-  title: 'Welcome to payments/next',
-  description: 'Generated by create-nx-workspace',
+  title: 'Firefox accounts',
+  description: 'Fireffox accounts',
```

Comment:
Typo: `Fireffox` -> `Firefox`

Reply:
Updated and added a TODO comment referencing [FXA-8227](https://mozilla-hub.atlassian.net/browse/FXA-8227)